tests/utils/test_maven.py

```diff
@@ -0,0 +1,119 @@
+"""Test for maven.py
+
+Test for the utility for maven operations.
+"""
+from testfixtures import TempDirectory
+from tests.helpers.base_tssc_test_case import BaseTSSCTestCase
+from tssc.utils.maven import generate_maven_settings
+
+
+class TestMavenUtils(BaseTSSCTestCase):
+    # def test_generate_maven_settings_maven_servers_is_none(self):
+
+    def test_generate_maven_servers_exist(self):
```

Comment:
Do you have a test for when servers are empty?

Reply:
@akim309 - passing this one to you

Reply:
@HunterGerlach in the other tests I set maven_servers to None so that it'll isolate whichever variable I'm testing and so I thought it was covered but if you want to see a test where it tests for no servers I can add it in

Reply:
Your tests currently cover the step properly. You can ignore this request.